cypress_shared/pages/apply/attachDocumentsPage.ts

```diff
@@ -29,6 +30,28 @@ export default class AttachDocumentsPage extends Page {
     })
   }
 
+  shouldBeAbleToDownloadDocuments() {
+    this.documents.forEach(document => {
+      // This is a hack to stop `cy.click()` from waiting for a page load
+      // See: https://github.com/cypress-io/cypress/issues/14857
+      cy.window()
+        .document()
+        .then(doc => {
+          doc.addEventListener('click', () => {
+            setTimeout(() => {
+              doc.location.reload()
+            }, 300)
+          })
+          cy.get(`a[data-cy-documentId="${document.id}"]`).click()
+        })
+
+      const downloadsFolder = Cypress.config('downloadsFolder')
+      const downloadedFilename = path.join(downloadsFolder, document.fileName)
+
+      cy.readFile(downloadedFilename, 'binary', { timeout: 15000 })
```

Comment:
You may have already done this but could we start with a smaller value and increase it if it takes longer?

Reply:
Good call 👍